stl/inc/xmemory

```diff
@@ -73,13 +73,28 @@ _INLINE_VAR constexpr size_t _New_alignof = (_STD max)(alignof(_Ty),
 
 // STRUCT _Default_allocate_traits
 struct _Default_allocate_traits {
+#ifdef __clang__ // TRANSITION, MSVC support for constexpr dynamic allocation
+    _CONSTEXPR20_DYNALLOC
+#endif // __clang__
     __declspec(allocator) static void* _Allocate(const size_t _Bytes) {
         return ::operator new(_Bytes);
     }
 
 #ifdef __cpp_aligned_new
+#ifdef __clang__ // TRANSITION, MSVC support for constexpr dynamic allocation

```

Comment:
The transition comment  below is different than this one, but it describes the same feature request for MSVC, so we should use the same comment here and abve